site.ts

```diff
@@ -225,7 +224,7 @@ export function inject({ config, posthog }) {
     const callSurveys = (posthog) => {
         posthog?.getActiveMatchingSurveys((surveys) => {
             surveys.forEach((survey) => {
-                if (sessionStorage.getItem("PostHogSurveyOnScreen") === "false") {
```

Comment:
sessionStorage stays as true here on refresh but the survey gets unloaded, best to just check if any survey literally exists on screen or not